app/(storyblok)/[[...slug]]/page.tsx

```diff
@@ -0,0 +1,116 @@
+import { Metadata } from 'next';
+import {
+  getStoryblokApi,
+  ISbStoriesParams,
+  StoryblokClient,
+} from '@storyblok/react/rsc';
+import StoryblokStory from '@storyblok/react/story';
+import { resolveRelations } from '@/utilities/resolveRelations';
+import { getPageMetadata } from '@/utilities/getPageMetadata';
+import { notFound } from 'next/navigation';
+
+const activeEnv = process.env.NODE_ENV || 'development';
+
+type PathsType = {
+  slug: string[];
+};
+
+type ParamsType = {
+  slug: string[];
+};
+
+// Control what happens when a dynamic segment is visited that was not generated with generateStaticParams.
+export const dynamicParams = false;
```

Comment:
Setting this to `true` will try to render paths that are not statically generated by fetching the data from Storyblok. Let's wait until Netlify fixes the plugin before we go down that route. Leaving this to false will cause any path not statically generated to 404.